UI/src/view/point_asset/trafficSignForm.js

```diff
@@ -75,6 +75,11 @@
 
     this.boxEvents = function(rootElement, selectedAsset, localizedTexts, authorizationPolicy, roadCollection, collection) {
 
+      rootElement.find('.form-traffic-sign').on('change', function(event) {
+          // force the field to be filled
+          selectedAsset.setPropertyByPublicId('opposite_side_sign', '0');
+      });
+
```

Comment:
Does this need to be forced? I don't think it should be mandatory.

Reply:
Yes, this field need to be forced.
Otherwise we can't update the traffic signs with a single choice with value ""